lib/waterdrop/config.rb

```diff
@@ -13,11 +13,11 @@ class Config
       # emit librdkafka statistics every five seconds. This is used in instrumentation.
       # When disabled, part of metrics will not be published and available.
       'statistics.interval.ms': 5_000,
-      # We set it to a value that is lower than `max_wait_time` to have a final verdict upon sync
-      # delivery
-      'message.timeout.ms': 55_000,
+      # We set it to a value that is lower than `max_wait_timeout` to have a final verdict upon
+      # sync delivery
+      'message.timeout.ms': 50_000,
       # Must be less or equal to `message.timeout.ms` defaults
-      'transaction.timeout.ms': 45_000
+      'transaction.timeout.ms': 55_000
```

Comment:
The comment states it should be `<= message.timeout.ms`. Right now, we have `55_000 <= 50_000`.